internal/interfaceadapter/repository/userrepo/repository.go

```diff
@@ -37,19 +37,19 @@ func (r *repository) GetUsers(ctx context.Context, IDs []uint64) (map[uint64]ent
 	  LEFT JOIN top_users t using(user_id)
 	  WHERE user_id IN (%s)
 	`
-	args := make([]interface{}, 0, len(IDs))
-	for _, id := range IDs {
+	args := make([]interface{}, 0, len(ids))
+	for _, id := range ids {
 		args = append(args, id)
 	}
-	placeholder := strings.Join(strings.Split(strings.Repeat("?", len(IDs)), ""), ",")
+	placeholder := strings.Join(strings.Split(strings.Repeat("?", len(ids)), ""), ",")
 	query := fmt.Sprintf(queryTemplate, placeholder)
 	rows, err := r.connManager.GetSlave().QueryContext(ctx, query, args...)
 	if err != nil {
 		return nil, fmt.Errorf("db access error: %w", err)
 	}
 	defer rows.Close()
 
-	userIDToEntity := make(map[uint64]entity.User)
+	userIDToEntity := make(map[uint64]*entity.User)
```

Comment:
userEntitiesぐらいでも良さそう
```suggestion
	userEntities := make(map[uint64]*entity.User)
```

Reply:
こちらのほうが分かりやすいので修正しました！